libnmstate/nm/active_connection.py

```diff
@@ -108,13 +105,9 @@ def _deactivate_connection_callback(self, src_object, result, user_data):
 
         try:
             success = src_object.deactivate_connection_finish(result)
-        except Exception as e:
-            if (
-                isinstance(e, GLib.GError)
-                # pylint: disable=no-member
-                and e.domain == NM_MANAGER_ERROR_DOMAIN
-                and e.code == NM.ManagerError.CONNECTIONNOTACTIVE
-                # pylint: enable=no-member
+        except GLib.Error as e:
```

Comment:
This now changes the `else` to be included under this type as well. Is that indeed the case?

Reply:
Yes, the exception must be always a `GLib.Error` one, the else is only handling other `GLib.Error exceptions`. If we get any other kind of exception here then it is a bug from libnm.

Reply:
That is fine, but you do not cleanup on such a failure. It will just raise an unknown exception, the handler will not get disconnected, etc...

Reply:
Done, thanks!